bom/pom.xml

```diff
@@ -50,6 +50,7 @@
     <truth.version>0.28</truth.version>
     <vertx.version>3.3.3</vertx.version>
     <vertx-proton.version>3.3.3</vertx-proton.version>
+    <vertx-mqtt.version>3.4.0-SNAPSHOT</vertx-mqtt.version>
```

Comment:
We have already talked about this. We should try to get a "milestone" release of this dependency as quickly as possible and create a CQ with Eclipse for it then ...